server/graphql/v2/object/Order.js

```diff
@@ -184,7 +192,7 @@ export const Order = new GraphQLObjectType({
         },
       },
       hostFeePercent: {
-        type: GraphQLInt,
+        type: GraphQLFloat,
```

Comment:
Could you go extra mile and fix all occurrences of `hostFeePercent` to `GraphQLFloat`